UIAutomationTests/readme.md

```diff
@@ -0,0 +1,16 @@
+# UI Automation Tests
+
+## Goals
+Provides a way of automatically testing the Exasol PowerBI Connector
+- It can be run from the .sln file using Test -> Run All Tests.
```

Comment:
```suggestion
- It can be run from the `.sln` file using `Test -> Run All Tests`.
```
```suggestion
- It can be run from the .sln file using Test -> Run All Tests.
```